Update index.hbs

nemolivier opened the pull request:
Je suis quasi-certain que le hongrois et l’allemand ne sont pas bons (pas de référence aux 7 et 8).
Et j’ai l’impression qu’il manque encore des langues par rapport à celles du pad https://titanpad.com/l9dvbO5S8U :)

nemolivier committed:
Update index.hbs

Je suis quasi-certain que le hongrois et l’allemand ne sont pas bons (pas de référence aux 7 et 8).

tomplays commented:
y a des fautes sur les liens du genre :    <a href="/de/{{callSlugs.el}}.html">Voir l'appel</a>

tomplays commented:
<a \\h\\\ref="/de/{{callSlugs.el}}.html">Voir l'appel</\\\a>

tomplays commented:
fautes sur liens type : /de/{{callSlugs.el}}

nemolivier committed:
Update index.hbs

Correction (le hr c’est du croate…) et j’ai passé tous les noms de langue en Anglais (le grec était sur ce modèle là). Je re-changerai si ça ne vous paraît pas pertinent.
